architecture/2024-10-deploymenttemplate-controller.md

```diff
@@ -0,0 +1,421 @@
+# Radius DeploymentTemplate Controller
+
+* **Author**: Will Smith (@willdavsmith)
+
+## Overview
+
+Today, users of Radius and future adopters of Radius use many Kubernetes-specific tools in their production workflows. Most of these tools operate on Kubernetes resources exclusively - which presents a problem when trying to deploy resources defined in Bicep manifests. This design proposes the creation of a new Kubernetes controller in Radius called the DeploymentTemplate Controller that will allow users to deploy resources defined in Bicep manifests using Kubernetes tooling using Radius.
+
+## Today's Status
+
+Today, Radius users are unable to deploy resources defined in Bicep manifests using Kubernetes tooling. This is because Kubernetes tools do not understand Bicep manifests. Users must use the Radius CLI to deploy resources defined in Bicep manifests.
```

Comment:
This actually opens a new door for almost all users who deploy resources using Bicep files on to Kubernetes clusters.